WcaOnRails/app/controllers/api/v0/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Api::V0::UsersController < Api::V0::ApiController
+  def token
+    if current_user
+      render json: { status: "ok" }
+    else
+      render status: :unauthorized, json: { error: "Please log in" }
```

Comment:
Do we have the "Please log in" (or a similar, equivalent phrasing) somewhere in our i18n file?
I feel like some translation should exist already.

Reply:
we do not have one, I added one though, because I can then use it in the other PRs as well